<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1255</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page_1">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="uploadButton">
              <property name="text">
               <string>UPLOAD</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="createDirectoryButton">
              <property name="text">
               <string>CREATE NEW DIRECTORY</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="informationsButton">
              <property name="text">
               <string>INFORMATIONS</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLineEdit" name="filterLineEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="searchButton">
              <property name="text">
               <string>SEARCH</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox">
              <item>
               <property name="text">
                <string>Created ASC</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Created DES</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Name ASC</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Name DES</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QScrollArea" name="files">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="filesContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>1215</width>
               <height>502</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="statusLabel">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="backButton">
              <property name="text">
               <string>BACK</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>DELETE</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="downloadButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>DOWNLOAD</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="moveButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>RENAME / MOVE</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
